source/extensions/omni.isaac.lab/omni/isaac/lab/assets/articulation/articulation_data.py

```diff
@@ -256,14 +259,17 @@ def update(self, dt: float):
     def root_state_w(self):
         """Root state ``[pos, quat, lin_vel, ang_vel]`` in simulation world frame. Shape is (num_instances, 13).
 
-        The position and quaternion are of the articulation root's actor frame. Meanwhile, the linear and angular
-        velocities are of the articulation root's center of mass frame.
+        The position and quaternion are of the articulation root's actor frame relative to the world.
```

Comment:
NIT: Should we mention they are all now in the same reference frame? Might reword as I'm not sure my change fully makes sense :smile: 
```suggestion
        The position, quaternion and linear / angular velocities are all in the articulation root's actor frame relative to the world.
```